### PR TITLE
Started DSL project

### PR DESCRIPTION
**Pull request by refusal-type-bamboo**
Started implementing the parser for layouts.
Layout initialization not added.

### CODE REVIEW

**Review by titzer — APPROVED** [review]
Looks good, can you bump the version number with 'aeneas bump'?

**Comment by refusal-type-bamboo** [comment]
Updated. Sorry for that!

File: aeneas/src/vst/Parser.v3 [hunk 1]
```diff
@@ -132,6 +132,14 @@ component Parser {
 					return true;
 				}
 			}
+			'l' => {
```
Comment (titzer) [inline aeneas/src/vst/Parser.v3:135]: I had a look back at our whiteboard conversation, and I think we should use the 'struct' keyword for fixed layouts. Can you update this PR to use 'struct' for the keyword and name the new Vst node `VstStruct`?

**Review by titzer — CHANGES_REQUESTED** [review]

File: aeneas/src/vst/Parser.v3 (hunk 1 above)
Comment (refusal-type-bamboo) [reply aeneas/src/vst/Parser.v3]: Got it. All keyword 'layout' changed to 'struct', methods renamed.

**Review by titzer — APPROVED** [review]
lgtm, thanks!

**Pull request merged**